src/server/adyenMiddleware.ts

```diff
@@ -107,35 +107,33 @@ export const handleNewAdyen3dsPostRedirect: Router.IMiddleware<
       },
     })
 
-    console.log('result', result)
-
     if (result.status !== 200) {
       throw new Error(
         `Expected status code from Graphql endpoint to be 200 when submitting adyen redirection, but was really ${result.status}`,
       )
     }
 
     if (
-      ['Authorised', 'Pending'].includes(
+      ['AUTHORISED', 'PENDING'].includes(
         result.data?.data?.paymentConnection_submitAdyenRedirection?.status,
       )
     ) {
       ctx.redirect(
         ctx.params.locale
-          ? `/${ctx.params.locale}/new-member/download`
-          : '/new-member/connect-download',
+          ? `/${ctx.params.locale}/new-member/checkout/payment/${quoteCartId}/true`
```

Comment:
Hmm unsure if this is the best URL to use -- what does "true" mean?

Reply:
yeah, was planning to change this at a later date but lacked a better name. guess ' 3dssuccess ' would be better.